onnxruntime/python/tools/transformers/fusion_rotary_attention.py

```diff
@@ -388,6 +440,24 @@ def fuse(self, normalize_node, input_name_to_nodes, output_name_to_node):
             transpose_v, reshape_v, matmul_v = v_nodes_3
             v_nodes = v_nodes_3
             present_v = transpose_v.output[0]
+        elif v_nodes_4 is not None and len(v_nodes_4) == 9:
+            logger.debug('fuse_rotary_attention: v_nodes_4')
+            logger.debug('*' * 30)
```

Comment:
We should use `logger.debug` when the pattern matching fails. These messages will print when pattern matching is successful.

Reply:
This is for debug convenience, since these debug message will only be printed when we enable verbose I think it is okay to keep them